models/platform-limits/README.md

```diff
@@ -0,0 +1,17 @@
+# @commercetools-test-data/platform-limits
+
+This package provides the data model for the commercetools platform `Limits` type
+
+https://docs.commercetools.com/api/limits
+
+# Install
+
+```bash
+$ pnpm add -D @commercetools-test-data/platform-limits
+```
+
+# Usage
+
+```ts
+
+```
```

Comment:
Readme is updated in the [part-3 PR](https://github.com/commercetools/test-data/pull/657)